spec/readers/datacite_reader_spec.rb

```diff
@@ -1653,4 +1653,12 @@
     )
   end
 
+  it "Parses dates with fractional seconds" do
```

Comment:
Can we add a new test as well, that also handles the ranges we talked about.
Simplest way might be to make the fixture actually just something like datacite-example-dates.xml
and then have both a fractional date in it and also a date with a range.
then add some expects for both.